cloud/terraform/aws/dependencies/data.tf

```diff
@@ -5,4 +5,13 @@ data "aws_eks_cluster" "cluster" {
 }
 data "aws_eks_cluster_auth" "cluster_auth" {
 	name=var.cluster_name
+}
+data "aws_subnets" "eks_subnets" {
+	filter {
+		name="vpc-id"
+		values=[data.aws_eks_cluster.cluster.vpc_config[0].vpc_id]
```

Comment:
Should `data.aws_eks_cluster.cluster.vpc_config[0]` be a local since we use it several times? Not sure what I think myself.